exercises/allergies/canonical-data.json

```diff
@@ -72,76 +78,109 @@
         {
           "description": "no allergies at all",
           "property": "list",
-          "score": 0,
+          "input": {
+            "score": 0
+          },
           "expected": []
         },
         {
           "description": "allergic to just eggs",
           "property": "list",
-          "score": 1,
-          "expected": ["eggs"]
+          "input": {
+            "score": 1
+          },
+          "expected": [
+            "eggs"
+          ]
         },
         {
           "description": "allergic to just peanuts",
           "property": "list",
-          "score": 2,
-          "expected": ["peanuts"]
+          "input": {
+            "score": 2
+          },
+          "expected": [
```

Comment:
Do the "expected"'s need to be changed or just the the addition of "input"?  I don't know that the present "expected" entries violate the schema.

Reply:
#996 Just mentions a change to `input`, I think `expected` is always one variable.